etc/test-helper.js

```diff
@@ -59,7 +59,7 @@ function checkChannels(expectedChannels, actualChannels) {
   }
 }
 
-function areSetsEqual(set1, set2) {
+function areUnorderedListsEqual(set1, set2) {
```

Comment:
This change is unrelated, but I felt it was more accurate since the collections in question may contain duplicate items.